edgexfoundry/edgex-go
# feat: Remove ZeroMQ MessageBus capability

closes #4233

BREAKING CHANGE: ZeroMQ MessageBus capability no longer available
 
Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Build local docker images from this branch
Run the docker secure stack using `make run dev ds-virtual`
Verify all core/support/secure services running as expected. 

## New Dependency Instructions (If applicable)
